src/node/transaction.cpp

```diff
@@ -14,26 +14,30 @@
 
 #include <future>
 
-TransactionError BroadcastTransaction(const CTransactionRef tx, uint256& hashTx, std::string& err_string, const CAmount& highfee)
+TransactionError BroadcastTransaction(const CTransactionRef tx, std::string& err_string, const CAmount& max_tx_fee, bool relay, bool wait_callback)
 {
+    assert(g_connman);
```

Comment:
nit: I'd prefer to see a comment next to this assert to explain why `g_connman` must always be assigned here.

Reply:
Should be part of #16503, maybe